desktop-app/src/main/main.ts

```diff
@@ -119,7 +125,9 @@ const createWindow = async () => {
     width,
     height,
     icon: getAssetPath('icon.png'),
+    titleBarStyle: 'hidden',
```

Comment:
Also, we want to ensure this doesn't affect Mac and Linux versions of the app, do you have any ideas how it works there after this change?

Reply:
I don't think this would affect Mac/Linux versions, as that option disables default taskbar styles and results in a full-size content window.

Reply:
I just tried this code on Mac and it affects the title bar color.

Preference disabled(default):
<img width="1307" alt="Screenshot 2023-10-29 at 11 39 21 AM" src="https://github.com/responsively-org/responsively-app/assets/1283424/ed1286d9-ecea-471d-bda5-6be58e75d79a">

Preference enabled:
<img width="1193" alt="Screenshot 2023-10-29 at 11 38 42 AM" src="https://github.com/responsively-org/responsively-app/assets/1283424/f45d1e51-49cd-433f-8968-5a04e673e13e">


Is it possible to selectively apply this by checking, only when the platform is windows?

Reply:
Try moving the styles to a new css file and optionally import based on UA data?
<img width="628" alt="Screenshot 2023-10-29 at 11 53 53 AM" src="https://github.com/responsively-org/responsively-app/assets/1283424/287ca128-7dac-4e72-a018-e275ce75e890">